backend/app/api/endpoints/cloud_projects.py

```
@@ -80,9 +83,9 @@ def _project_response(
 def create_cloud_project(
     values: CloudProjectCreate,
     db: Session = Depends(get_db),
-    current_user: User = Depends(get_current_user_flexible_for_executor),
+    current_user: User = Depends(get_current_user_jwt_apikey_tasktoken),
```

Ruff (0.16.3)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)